### ISSUE TITLE
Node.js v12.18.1 on armv6, armv7 and armv8

### ISSUE THREAD

**Issue by n1ru4l** [robertsLando/pkg-binaries#7]
Would it be possible to also build binaries for Node.js v12.18.1 for armv6, armv7, and armv8?

**Comment by robertsLando** [comment]
They are there. Remember that armv6 can also be used in armv7 and armv8 is equal to arm64

---

Daniel

**Comment by n1ru4l** [comment]
@robertsLando Thanks for the clarification and also for the binaries 👍

**Comment by robertsLando** [comment]
@n1ru4l You welcome :)

**Comment by n1ru4l** [comment]
@robertsLando So we are finally trying to upgrade to Node12 now and we run into a few complications.

We are using docker buildx for creating armv7 binaries (since armv6 is not available as a docker image) and [not supported by debian](https://github.com/nodejs/docker-node/issues/1294) and dropped by [Node.js](https://github.com/nodejs/build/issues/1677)).

This means without a `fetched-v12.18.1-linux-armv7` we are unable to build those.

Could you please consider providing binaries for armv7?

**Comment by robertsLando** [comment]
Your edit is the answer to your question :) Once that is sorted I will update the build script, actually if you chech how I build now you will see I select armv6 even for armv7 build on purpose to fix that

**Comment by n1ru4l** [comment]
@robertsLando Yeah I actually figured that out already 😅 So that means it should actually work today if I would place `fetched-v12.18.1-linux-armv6` as `fetched-v12.18.1-linux-armv7` in the cache, is that correct?

**Comment by robertsLando** [comment]
@n1ru4l Unfortunally I rarely see a PR to be merged or also considered with both pkg or pgk-fetch, if you don't see any response I could consider to use your branch to build pkg

**Comment by robertsLando** [comment]
> So that means it should actually work today if I would place fetched-v12.18.1-linux-armv6 as fetched-v12.18.1-linux-armv7 in the cache, is that correct

That is what I'm doing in last year and I didn't have any problem. Using armv6 binaries in armv7 seems safe

**Comment by n1ru4l** [comment]
Yeah I misunderstood that comment then 🤣 check https://github.com/robertsLando/pkg-binaries/pull/9
